fe/fe-core/src/main/java/com/starrocks/planner/OlapTableSink.java

```diff
@@ -488,19 +498,31 @@ private TOlapTableLocationParam createLocation(OlapTable table) throws UserExcep
 
         // check if disk capacity reach limit
         // this is for load process, so use high water mark to check
-        Status st = GlobalStateMgr.getCurrentSystemInfo().checkExceedDiskCapacityLimit(allBePathsMap, true);
-        if (!st.ok()) {
-            throw new DdlException(st.getErrorMsg());
+        if (!Config.use_staros) {
```

Comment:
`LakeSystemInfoService` gives a default implementation and no need to check `use_staros`